### ISSUE TITLE
[Combobox] Selected Elements Announced as Not Selected

### ISSUE THREAD

**Issue by DanielleRameau** [sl-design-system/components#2743]
### Provide a general summary of the issue here

**Problem:**
When using the combobox with multiple selection and grouped options, selected elements are incorrectly announced as not selected. This may be due to missing or incorrect ARIA attributes (e.g. aria-selected).

<img width="1490" height="592" alt="Image" src="https://github.com/user-attachments/assets/e1907656-2963-4997-af39-16ca6bcc24d0" />

**Action:**
Fix the accessibility attributes so selected items are correctly announced.

Issue found during internal a11y audit: https://github.com/sl-design-system/components/issues/2516#issuecomment-3356071993

### 👤 Your name

DR

### 🧢 Your Product/Team

SLDS

**Comment by a11ymiko** [comment]
This issue no longer exists in Combobox v0.1.7
